example/spec/parallel/moveGranuleCollections/move-granule-collection-spec-utils.js

```diff
@@ -0,0 +1,253 @@
+const {
+  promiseS3Upload,
+  deleteS3Object,
+} = require('@cumulus/aws-client/S3');
+const {
+  granules,
+  collections,
+} = require('@cumulus/api-client');
+
+const path = require('path');
+const fs = require('fs');
+const { constructCollectionId } = require('../../../../packages/message/Collections');
+
+const getTargetFiles = (targetUrlPrefix) => [
+  {
+    bucket: 'cumulus-test-sandbox-protected',
+    key: `${targetUrlPrefix}/MOD11A1.A2017200.h19v04.006.2017201090724.hdf`,
+  },
+  {
+    bucket: 'cumulus-test-sandbox-public',
+    key: `${targetUrlPrefix}/jpg/example2/MOD11A1.A2017200.h19v04.006.2017201090724_1.jpg`,
+  },
+  {
+    bucket: 'cumulus-test-sandbox-public',
+    key: `${targetUrlPrefix}/MOD11A1.A2017200.h19v04.006.2017201090724_2.jpg`,
+  },
+  {
+    bucket: 'cumulus-test-sandbox-public',
+    key: `${targetUrlPrefix}/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml`,
+  },
+];
+
+const getSourceCollection = (sourceUrlPrefix) => (
+  {
+    files: [
```

Comment:
This is part of the collection configuration. We ingest different granules/files to s3 to avoid conflicts.